internal/infrastructure/ceph/rbd.go

```diff
@@ -98,13 +109,43 @@ func (r *RBDRepository) ApplyDiffToBlockDevice(blockDevicePath, diffFilePath, fr
 func (r *RBDRepository) ApplyDiffToRawImage(
 	rawImageFilePath, diffFilePath, fromSnapName, toSnapName string, expansionUnitSize uint64,
 ) error {
-	diffFile, err := os.Open(diffFilePath)
+	// TODO: These parameters will be configurable via arguments in PR #187.
+	rawChecksumChunkSize := uint64(64 * 1024)        // 64 KiB
```

Comment:
```suggestion
	rawChecksumChunkSize := defaultRawChecksumChunkSize        // 64 KiB
```

In addition, it's better to define defaultDiffChecksumChunkSize too.